src/libraries/Microsoft.PowerFx.Core/Public/Config/ReadOnlySymbolTable.cs

```diff
@@ -439,7 +479,9 @@ internal class EnumerateNamesOptions
 
         DPath INameResolver.CurrentEntityPath => default;
 
-        bool INameResolver.SuggestUnqualifiedEnums => false;
+        bool INameResolver.SuggestUnqualifiedEnums => false;
+
+        IEnumerable<KeyValuePair<DName, FormulaType>> INameResolver.DefinedTypes => default;
```

Comment:
>  IEnumerable<KeyValuePair<DName, FormulaType>> INameResolver.DefinedTypes => default; [](http://example.com/codeflow?start=6&length=86)

do we need this?

These are just non-implemented ones . And we impl on line 363